containers/runner/run-kstest

```diff
@@ -93,8 +93,10 @@ ${KSTESTS_DIR}/scripts/log2json --scenario $SCENARIO --output $TEST_LOG.json $TE
 # Fixup permissions
 chmod -R a+rw /var/tmp/kstest-*
 # Clean up (artifacts from broken test)
-find /var/tmp/kstest-* -name "*.iso" -delete
-find /var/tmp/kstest-* -name "*.img" -delete
+if [ "$KSTESTS_KEEP" != "2" ]; then
+    find /var/tmp/kstest-* -name "*.iso" -delete
+    find /var/tmp/kstest-* -name "*.img" -delete
+fi
```

Comment:
Wouldn't this increase required size a lot for the artifacts? If yes, we shouldn't use this on GH Actions.

Reply:
IMO `$KSTESTS_KEEP` should default to 1 unless explicitly set to other value, so the disk images shouldn't be kept by default. I introduced this change because there was no way (unless I missed it) to keep the images, which may be crucial for debugging or writing new tests in some cases.